### PR TITLE
change mssql image and increase sleep timeout

### PR DESCRIPTION
**Pull request by vishal423**
-   Please make sure the below checklist is followed for Pull Requests.

-   [x] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/server/templates/src/main/docker/mssql.yml.ejs [hunk 1]
```diff
@@ -27,7 +27,7 @@ services:
             - ACCEPT_EULA=Y
             - SA_PASSWORD=yourStrong(!)Password
             - MSSQL_DATABASE=<%= baseName %>
-            - MSSQL_SLEEP=7
+            - MSSQL_SLEEP=60
```
Comment (DanielFran) [inline generators/server/templates/src/main/docker/mssql.yml.ejs]: Should we add the PID?  MSSQL_PID: "Developer" or  MSSQL_PID: "Express"

Comment (DanielFran) [reply generators/server/templates/src/main/docker/mssql.yml.ejs]: MSSQL_PID=<your_product_id | edition_name> (default: Developer)

Comment (vishal423) [reply generators/server/templates/src/main/docker/mssql.yml.ejs]: default is `Developer`. If we want to change, then, I will prefer `Express` as it's free from any license obligations.

Comment (pascalgrimaud) [reply generators/server/templates/src/main/docker/mssql.yml.ejs]: don't hesitate to open a new PR if needed :)

**Pull request merged**